src/main/java/org/folio/circulation/support/CqlQuery.java

```diff
@@ -29,6 +29,14 @@
     return Result.of(() -> new CqlQuery(format("%s==\"%s\"", index, value), none()));
   }
 
+  /**
+   * The expression `comparitorSymbol: '<>' ` for finding a non-empty value
+   * is not supported in the CqlQuery.
+   */
+  public static Result<CqlQuery> exactNotEmpty(String index) {
```

Comment:
I found the name of this confusing and misread the JavaDoc comment, thinking it stated the use of `<>`

Could this be clearer if it was changed to explicitly be a greater than comparison. For example, `greaterThan(String index, String value)`

If we want to be explicit about a non-empty check, then maybe we could add an alias like `notEmpty(String index)` that uses the greater than method. And provide JavaDoc something like `Uses greater than ('>'), as not equals operator ('<>') is not supported in CQL at present`

Also, please can a unit test be added for this.

What do you think?

Reply:
I will change the name of the method to `greaterThan(String index, String value)` and add a unit test for it.

Reply:
done